piperider_cli/configuration.py

```diff
@@ -165,3 +163,29 @@ def ask_for_datasource(self):
             ]
             answers = inquirer.prompt(questions, raise_keyboard_interrupt=True)
             return answers['datasource']
+
+
+def _load_dbt_profile(path):
+    from jinja2 import Environment, FileSystemLoader
+
+    def env_var(var):
```

Comment:
This configure is not working, because there is optional 2nd argument. see 
https://docs.getdbt.com/reference/dbt-jinja-functions/env_var

```
infuse_finance:
  outputs:
    dev:
      account: kb22750.us-central1.gcp
      database: INFUSE_FINANCE
      password: "{{ env_var('SNOWFLAKE_PASSWORD') }}"
      schema: PUBLIC
      threads: 4
      type: snowflake
      user: infuseaidev
      warehouse: "{{ env_var('SNOWFLAKE_WAREHOUSE', LOAD_WH) }}"
  target: dev
 ```